browser/brave_wallet/android/sources.gni

```diff
@@ -0,0 +1,7 @@
+brave_browser_brave_wallet_android_sources = [
```

Comment:
these files depend on ProfileManager, but the Profile should be passed in because this doesn't properly handle regular vs incognito

Reply:
https://github.com/brave/brave-browser/issues/18450